scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -181,11 +197,37 @@ sealed trait ReduceStep[K, V1] extends KeyedPipe[K] {
   def mapped: TypedPipe[(K, V1)]
   // make the pipe and group it, only here because it is common
   protected def groupOp[V2](gb: GroupBuilder => GroupBuilder): TypedPipe[(K, V2)] =
+    groupOpWithValueSort[V2](None)(gb)
+
+  protected def groupOpWithValueSort[V2](valueSort: Option[Ordering[_ >: V1]] = None)(gb: GroupBuilder => GroupBuilder): TypedPipe[(K, V2)] =
```

Comment:
we never use it with the default arg, can we not use the default arg at all to make it more clear?